spec/models/miq_request_task/dumping_spec.rb

```diff
@@ -0,0 +1,53 @@
+describe MiqRequestTask do
+  context "::Dumping" do
+    let(:task) { FactoryGirl.create(:miq_request_task) }
+
+    describe '#dump_obj' do
+      it "calls .dump_obj" do
+        expect(task.class).to receive(:dump_obj)
+        task.dump_obj({:param_1 => 1})
+      end
+
+      it 'hides passwords' do
+        expect($log).to receive(:info).with(/<PROTECTED>/)
+        task.dump_obj({:my_password => "secret"}, "my choices: ", $log, :info, :protected => {:path => /[Pp]assword/})
+      end
+    end
+
+    describe '.dump_obj' do
+      it 'accepts a hash' do
+        expect(task.class).to receive(:dump_hash)
+        task.class.dump_obj({:param_1 => 1}) {}
```

Comment:
You do not need the curly braces here and there is an extra set of curly braces after the method call that needs to be removed.